torch/nn/modules/loss.py

```diff
@@ -151,9 +151,15 @@ class NLLLoss(_WeightedLoss):
         - Input: :math:`(N, C)` where `C = number of classes`, or
             :math:`(N, C, d_1, d_2, ..., d_K)` with :math:`K \geq 2`
             in the case of `K`-dimensional loss.
+            Eg:  [0,0,1]
+
         - Target: :math:`(N)` where each value is :math:`0 \leq \text{targets}[i] \leq C-1`, or
             :math:`(N, d_1, d_2, ..., d_K)` with :math:`K \geq 2` in the case of
             K-dimensional loss.
+            Eg:[2]
```

Comment:
This should probably be a warning somewhere. You can use the `warning` directive:
```
.. warning::
    The target value should be a value indicating what class it belongs to. This is semantically different from the input, which contains log-probabilities for each class.
```